src/Sniff/ForbiddenSuperGlobalSniff.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ShopSys\CodingStandards\Sniff;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Inspired by
+ * @see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/master/WordPress/Sniffs/VIP/SuperGlobalInputUsageSniff.php
+ */
+final class ForbiddenSuperGlobalSniff implements Sniff
+{
+    /**
+     * @var string[]
+     */
+    private $superglobalVariables = ['$_COOKIE', '$_GET', '$_FILES', '$_POST', '$_REQUEST', '$_SERVER'];
+
+    /**
+     * @return int[]
```

Comment:
Why do not you use the PHP7 typehint here?

Reply:
I missed that, nice catch